src/pages/components/callout-quote.mdx

```diff
@@ -23,9 +24,22 @@ The Callout &mdash; quote pattern uses high contrast colors to create a delightf
 
 ![Default](../../images/pattern/callout-quote/callout-quote.jpg)
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Paragraph | Long copy | Yes   | 1 | LL (200 / 260)|
+| Source (ex. name)  | Short copy | No  | 1 | XS (65 / 85) |
+| Copy 1 (ex. role)  | Short copy | No | 1 | XS (65 / 85) |
+| Copy 2 (ex. company)  | Short copy | No | 1 | XS (65 / 85)   |
+| Quotation mark type | Option list | Yes | - | Double curved (default)<br/>Single curved<br/>Double angle<br/> Single angle<br/> Low high reversed double curved |
```

Comment:
These fall into a Quote component dependency if you wanted to link it out and maintain in one place.

[`/components/quote`](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/quote)

Reply:
@photodow So my thoughts here would be that technically we should just link out to "quote", but that's not super helpful to users. Cause in that case, my thinking would be to put Callout - Quote and quote under the same component since they have almost the same exact content model.

Do you think I should just add something in there that mentions the "quote" component?

Reply:
Mentioning it doesn't solve the problem of having to maintain it in multiple places. I won't let this block it, but I'm more concerned about if something happens to quote now we have to go back and change all the components using it. Side topic as well it's not clear when we link and when we don't because sometimes we do.

Reply:
adding a note about this to address later.